test/TestProjects/Models-Cadl/Generated/FixedIntEnum.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace ModelsInCadl
+{
+    /// <summary> Fixed int enum. </summary>
+    public enum FixedIntEnum
+    {
+        /// <summary> 1. </summary>
+        One,
```

Comment:
If we still keep serialization class, one thing we could improve for int value is that we could set the value for enum value, like `One = 1,`.

Reply:
Added int value assignment, so that the enum values are consistent with the definition.